### PR TITLE
thorntail-runner post

### PR DESCRIPTION
**Pull request by michalszynkiewicz**

### CODE REVIEW

File: src/posts/thorntail-runner.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+author: The Thorntail Team
+layout: post.jade
+---
+
+== Thorntail Runner
+
+Since 2.2.1.Final, there is a new way of running Thorntail apps in the IDE.
+
+Previously, the only way was running the `main` method of the `Swarm` class. It caused problems with classpath.
+Flat classpath that was used in this case didn't mimic the jboss modules classpath that is used in the `java -jar ...` execution well enough.
```
Comment (Ladicek) [inline src/posts/thorntail-runner.adoc]: "jboss modules" -> "JBoss Modules"

File: src/posts/thorntail-runner.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Thorntail Runner uses jboss modules approach. It was tested with all the issues users reported in the old way of running apps in the IDE and it solved all of them.
+
+=== How to use Runner
+
+==== Make it visible for your IDE
+In order to use Thorntail Runner, you have add `io.thorntail:thorntail-runner` to your application.
+If you use Maven, you can simply make your project depend on it.
+Just add the following to your dependencies.
+Don't worry, neither the thorntail-runner itself nor its dependencies will leak to the über jar that thorntail maven plugin generates for your project.
```
Comment (Ladicek) [inline src/posts/thorntail-runner.adoc]: I'd avoid the umlaut, just use "uberjar".

File: src/posts/thorntail-runner.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  <groupId>io.thorntail</groupId>
+  <artifactId>thorntail-runner</artifactId>
+  <version>${version.thorntail}</version>
+</dependency>
+
+----
+
+==== Run it
+If you use IntelliJ IDEA, use Ctrl/Cmd+N to find the `Runner` class. In Eclipse you can use Ctrl/Cmd+Shift+T.
+When you find it, it's just a matter of running it's `main` method.
```
Comment (Ladicek) [inline src/posts/thorntail-runner.adoc]: "it's" -> "its"

**Comment by Ladicek** [comment]
Maybe mention that Runner currently only works with WAR projects.

**Comment by michalszynkiewicz** [comment]
@Ladicek done. Thanks for the review!

File: src/posts/thorntail-runner.adoc [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Please note that it works only with `war` packaging.
+
+=== How to use Runner
+
+==== Make it visible for your IDE
+In order to use Thorntail Runner, you have add `io.thorntail:thorntail-runner` to your application.
+If you use Maven, you can simply make your project depend on it.
+Just add the following to your dependencies.
+Don't worry, neither the thorntail-runner itself nor its dependencies will leak to the uberjar that thorntail maven plugin generates for your project.
```
Comment (Ladicek) [inline src/posts/thorntail-runner.adoc]:
```suggestion
Don't worry, neither the `thorntail-runner` itself nor its dependencies will leak to the uberjar that Thorntail Maven plugin generates for your project.
```

File: src/posts/thorntail-runner.adoc [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+.pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>io.thorntail</groupId>
+  <artifactId>thorntail-runner</artifactId>
+  <version>${version.thorntail}</version>
+</dependency>
+
```
Comment (Ladicek) [inline src/posts/thorntail-runner.adoc]:
```suggestion
```

**Review by Ladicek — APPROVED** [review]

**Pull request merged**